internal/addon/addon.go

```diff
@@ -94,6 +95,24 @@ func AgentHealthProber() *agent.HealthProber {
 						},
 					},
 				},
+				{
+					ResourceIdentifier: workv1.ResourceIdentifier{
+						Group:    uiplugin.GroupVersion.Group,
+						Resource: UIPluginsResource,
+						Name:     "monitoring",
```

Comment:
Is the name usually advised for the incident detection plugin? I'm mainly thinking of what will this look like once Metrics introduce their own uiplugin. Are we at risk of having conflicting names?

Reply:
As far as I know the monitoring plugin (type=monitoring requires `monitoring` name) is a singleton so hopefully there should not be any conflicts.